media-sound/ladish/ladish-1.1.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+PYTHON_REQ_USE='threads(+)'
+
+inherit python-single-r1 waf-utils
+
+DESCRIPTION="LADI Session Handler - a session management system for JACK applications"
+HOMEPAGE="https://ladish.org"
+
+SRC_URI="https://dl.ladish.org/ladish/${P}-g36c489e4.tar.bz2"
+S="${WORKDIR}/${P}-g36c489e4"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+
+IUSE="debug +lash"
+
+BDEPEND="
+	virtual/pkgconfig
+"
+RDEPEND="
+	virtual/jack
+	media-sound/jack2[dbus]
+	sys-apps/dbus
+	dev-libs/expat
+	${PYTHON_DEPS}
+	lash? ( !!media-sound/lash )
+"
+DEPEND="${RDEPEND}"
+
+DOCS=( AUTHORS NEWS README.adoc )
```

Comment:
FYI: I think you could drop this line. portage usually installs a default set of documentation files and this should include all of the above. But needs to be tested.

Reply:
The source distribution contains files named README-docinfo*.html that are not to be installed. At least with ladish-1.1 tarball and EAPI=8, removing DOCS line will cause the two HTML files to be installed while they shouldn't. Upstream wise, I'll try to move these two files out of the root directory, at least in the release tarballs.